extensions/applicationinsights-analytics-js/src/JavaScriptSDK/AnalyticsPlugin.ts

```diff
@@ -265,6 +266,12 @@ export class AnalyticsPlugin extends BaseTelemetryPlugin implements IAppInsights
                     let inPv = pageView || {};
                     _pageViewManager.trackPageView(inPv, {...inPv.properties, ...inPv.measurements, ...customProperties});
         
+                    // Reset ajax attempts counter for the new page view
```

Comment:
Make this a helper function and call it where the code currently sets _trackAjaxAttempts to zero, removing the local _trackAjaxAttempts  variable in the process

Reply:
No revert the last change and only make the change for the helper to this file not in the ajax.ts